pkg/web/app.go

```diff
@@ -0,0 +1,40 @@
+package web
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/labstack/echo/v5"
+	"github.com/pocketbase/pocketbase"
+	"github.com/pocketbase/pocketbase/apis"
+	"github.com/pocketbase/pocketbase/core"
+)
+
+func App() *pocketbase.PocketBase {
+	app := pocketbase.New()
+
+	app.OnBeforeServe().Add(func(e *core.ServeEvent) error {
+		e.Router.Use(LoadAuthContextFromCookie(app))
+		e.Router.GET("/*", apis.StaticDirectoryHandler(os.DirFS("./pb_public"), false))
+		e.Router.POST("/login/", LoginHandler(app))
+		e.Router.GET("/logout/", LogoutHandler)
+		e.Router.GET("/", RootHandler)
+		return nil
+	})
+
+	return app
+}
+
+func RootHandler(c echo.Context) error {
+	info := apis.RequestInfo(c)
+	admin := info.Admin       // nil if not authenticated as admin
+	record := info.AuthRecord // nil if not authenticated as regular auth record
+
+	isGuest := admin == nil && record == nil
```

Comment:
If we set isGuest in the context we can fetch it here and check if the value is true or false. `v`can be renamed to something more descriptive 
```suggestion
      isGuest := c.Get(ContextAuthIsGuestKey)
	v := cast.ToBool(isGuest)
	if v == true {
		return c.HTML(http.StatusOK, "not logged in")
	}
```

Reply:
The C.get then cast into bool could be a helper func in the future if needed.

Reply:
Thank you for the feedback! I've pushed new changes now.